mlkem/asm/asm.h

```diff
@@ -16,18 +16,60 @@ void intt_asm_opt(int16_t *);
 void poly_reduce_asm_clean(int16_t *);
 void poly_reduce_asm_opt(int16_t *);
 
+void poly_tomont_asm_clean(int16_t *);
+void poly_tomont_asm_opt(int16_t *);
+
+void poly_mulcache_compute_asm_clean(int16_t *, int16_t *, int16_t *, int16_t *);
+void poly_mulcache_compute_asm_opt(int16_t *, int16_t *, int16_t *, int16_t *);
+
+void poly_tobytes_asm_clean(uint8_t *r, const int16_t *a);
+
+void polyvec_basemul_acc_montgomery_cached_asm_k2_clean(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+void polyvec_basemul_acc_montgomery_cached_asm_k3_clean(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+void polyvec_basemul_acc_montgomery_cached_asm_k4_clean(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+
+void polyvec_basemul_acc_montgomery_cached_asm_k2_opt(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+void polyvec_basemul_acc_montgomery_cached_asm_k3_opt(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+void polyvec_basemul_acc_montgomery_cached_asm_k4_opt(
+    int16_t *r, const int16_t *a, const int16_t *b, const int16_t *b_cache);
+
```

Comment:
Since you are using pre-processor conditionals to only include the corresponding parameter set in `polyvec_opt.S`, I don't think it is necessary to include the parameter set in the function name, is it? 
I'd prefer to keep it simpler here.

Reply:
Currently, the name is needed to disambiguate the functions to SLOTHY.